playwright/integrations.test.ts

```diff
@@ -35,11 +34,13 @@ describe("webhooks", () => {
     // page contains the url
     await expect(page).toHaveSelector(`text='${webhookReceiver.url}'`);
 
-    // --- go to tomorrow in the pro user's "30min"-event
-    const tomorrow = dayjs().add(1, "day");
-    const tomorrowFormatted = tomorrow.format("YYYY-MM-DDZZ");
+    // --- go to pro user's "30min"-event
+    await page.goto(`http://localhost:3000/pro/30min`);
 
-    await page.goto(`http://localhost:3000/pro/30min?date=${encodeURIComponent(tomorrowFormatted)}`);
+    // Click [data-testid="incrementMonth"]
+    await page.click('[data-testid="incrementMonth"]');
+    // Click [data-testid="day"]
+    await page.click('[data-testid="day"][data-disabled="false"]');
```

Comment:
Should we keep mine or yours? https://github.com/calendso/calendso/pull/1216/files#diff-74a4519f1ecec64c03189a71c5ab1662f4cefa3a18d700c2d120062575bdeb80R37-R41

Reply:
whatever makes it in first